src/backend/shortcuts/nonesteamgame/nonesteamgame.ts

```diff
@@ -173,17 +179,36 @@ function checkIfAlreadyAdded(object: Partial<ShortcutObject>, title: string) {
 
 /**
  * Adds a non-steam game to steam via editing shortcuts.vdf
- * @param steamUserdataDir Path to steam userdata directory
  * @param gameInfo @see GameInfo of the game to add
+ * @param steamUserdataDir Path to steam userdata directory, optional
+ * @param bkgDataUrl data url string for the background image, optional
+ * @param bigPicDataUrl data url of the Big Picture image, optional
  * @returns boolean
  */
 async function addNonSteamGame(props: {
-  steamUserdataDir: string
   gameInfo: GameInfo
-  bkgDataUrl: string
-  bigPicDataUrl: string
+  steamUserdataDir?: string
+  bkgDataUrl?: string
+  bigPicDataUrl?: string
```

Comment:
made these 3 props optional so we can pass them during tests but we don't need to pass those props outside of tests